### DISCUSSION TITLE
feat(getUniqueNames): new function

### DISCUSSION THREAD

**Discussion by carlosgamezdev** [vueuse/vueuse#1786] [Ideas]
Hi VueUse team,

I recently had the need to generate an arbitrary number of string identifiers (names) from a fixed alphabet (A,B,C,...).

Think of the column names for a typical spreadsheet: ```"A,B,C,D, ...., AA, AB, AC ... AAA, AAB, AAC, ..."```

I made a small function that returns this array of unique names.

Would there be interest in adding this to the Utilities section?

Cheers,
Carlos

**Comment by adarshmadrecha** [comment]
Wouldn't this lead eventually to adding all lodash functionalities to VueUse?

**Reply by carlosgamezdev** [reply]
Yep, maybe you're right. Interestingly, as part of this I made another small function that loops over a given array. I found out this had been implemented already in the useCycleList function.
